.changelog/2269.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_cloud_backup_snapshot_export_bucket: Calls DeleteExportBucket before checking for a status update. It avoids the delete operation to hang.
```

Comment:
```suggestion
resource/mongodbatlas_cloud_backup_snapshot_export_bucket: Calls DeleteExportBucket before checking for a status update so that the delete operation doesn't hang.
```